pkg/login/social/social.go

```diff
@@ -75,6 +75,7 @@ type OAuthInfo struct {
 	UsePKCE                 bool              `mapstructure:"use_pkce"`
 	UseRefreshToken         bool              `mapstructure:"use_refresh_token"`
 	Extra                   map[string]string `mapstructure:",remain"`
+	SignoutRedirectUrl      string   `toml:"signout_redirect_url"`
```

Comment:
Could you please change the tag to `mapstructure:"signout_redirect_url"` and move it above the `Extra` field?

Reply:
@mgyongyosi will take a look and align, incase of queries will reach out to you

Reply:
@mgyongyosi 
Modified as per the comment. continuous-integration/drone/pr is pending state. is it due to Milestone is set as 10.2.x?